app/src/main/java/org/sil/storyproducer/model/Phase.kt

```diff
@@ -3,146 +3,176 @@ package org.sil.storyproducer.model
 import org.sil.storyproducer.R
 import org.sil.storyproducer.controller.MainActivity
 import org.sil.storyproducer.controller.RegistrationActivity
-import org.sil.storyproducer.controller.export.CreateActivity
+import org.sil.storyproducer.controller.export.FinalizeActivity
 import org.sil.storyproducer.controller.export.ShareActivity
 import org.sil.storyproducer.controller.learn.LearnActivity
 import org.sil.storyproducer.controller.pager.PagerBaseActivity
 import org.sil.storyproducer.controller.remote.WholeStoryBackTranslationActivity
 
-
 enum class PhaseType {
-    WORKSPACE, REGISTRATION, STORY_LIST, LEARN, DRAFT, COMMUNITY_CHECK, CONSULTANT_CHECK, DRAMATIZATION, CREATE, SHARE, BACKT, WHOLE_STORY, REMOTE_CHECK
+    WORKSPACE,
+    REGISTRATION,
+    STORY_LIST,
+    LEARN,
+    TRANSLATE_REVISE,

```

Comment:
@jakedallinson  I think I see a problem here. Sorry.
A phaseType value is saved as part of the @JsonClass Story for member variable lastPhaseType.
I'm looking in story.json written by v3.0.4 and near the end I see   "lastPhaseType":"DRAFT"
So to maintain our product objective of reading existing projects written by prior SP versions, I think we have to decide 1) stay with the original ENUM values here, or 
2) we have to code a way to transform old values to new values, eg after the Story object is initialized from story.json we do
if lastPhaseType==DRAFT then lastPhaseType = TRANSLATE_REVISE
but actually you'd want to use a table of equivalent old and new enum values for those that changed.
Or 3) we ask Robin if the loss of the lastPhaseType is infrequent and minor enough that we just let it go! (Added code: If the lastPhaseType is not recognized, set it to LEARN)

Jake, it looks like a cornerstone of the nice cleanup you have done is to update these enum names, and this helps with clarity of the code. Option 2 would require some added migration code that would need to be tested and maintained for a time. Unit tests could do this. 
Let's ask Robin about option 3. What do you think?

Reply:
Yes I say we go with option 3! I can type up an email to Robin

Reply:
We have confirmation from Robin that option 3 is okay. We'll do that.
@jakedallinson Can you check that - When story.json is read in, if the lastPhaseType is not recognized, it gets set to LEARN